metrics/deploy/lib/deploy/deploy_command.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:args/command_runner.dart';
+import 'package:process_run/process_run.dart' as cmd;
+import 'package:process_run/shell.dart';
+import 'package:deploy/cli/firebase/firebase_command.dart';
+import 'package:deploy/cli/gcloud/gcloud_command.dart';
+import 'package:deploy/cli/git/git_command.dart';
+
+/// class extending [Command] to facilitate  building GCloud project,build and deploy metrics app.
+class DeployCommand extends Command {
+  @override
+  final name = "deploy";
+  @override
+  final description =
+      "Creates GCloud and Firebase project and deploy metrics app.";
+
+  final firebase = FirebaseCommand();
+  final gcloud = GCloudCommand();
+  final git = GitCommand();
+
+  DeployCommand();
+
+  @override
+  Future<void> run() async {
+    final firebaseToken = await _login();
+
+    final projectID = await gcloud.addProject();
+
+    await firebase.addFirebase(projectID, firebaseToken);
+
+    final region = await _selectRegion();
+
+    await gcloud.addProjectApp(region, projectID);
+
+    await gcloud.createDatabase(region, projectID);
+
+    final appID = await firebase.createWebApp(projectID, firebaseToken);
+    const repoURL = 'git@github.com:platform-platform/monorepo.git';
+    const srcPath = 'src';
```

Comment:
Can you please make these fields private static const members? 
To be something like this: 
```
class DeployCommand extends Command {
  static const _repoURL = 'git@github.com:platform-platform/monorepo.git';
  static const _srcPath = 'src';
```